webapp/app/forms/steps/eligibility_steps.py

```diff
@@ -761,23 +771,20 @@ def __init__(self, endpoint, stored_data=None, *args, **kwargs):
         
     def _main_handle(self):  
         super()._main_handle()
-              
-        self.render_info.additional_info['dependent_notes'] = get_dependent_notes(self)
+        
+        dependent_notes = []
+    
+        if data_fits_data_model_from_list([UserAElsterAccountEligibilityData, UserBElsterAccountEligibilityData], self.stored_data):
+            dependent_notes.append(_l('form.eligibility.result-note.user_b_elster_account-registration-maybe'))
```

Comment:
to me, the id of the message is a bit misleading. In my head it should be something like `form.eligibility.result-note.both_elster_account-registration-maybe`